copter/source/docs/traditional-helicopter-swashplate-setup.rst

```diff
@@ -155,9 +155,45 @@ that produces zero thrust. In Copter 3.6 and ealier, the minimum collective pitc
 could command was done with the parameter ``H_COL_LAND_MIN``. 
 
 Set :ref:`H_SV_MAN <H_SV_MAN>` to 3 to have the swashplate move to the mid position.  Adjust :ref:`H_COL_MID <H_COL_MID>` until the blade
-pitch angle measures the desired collective blade pitch that corresponds to zero thrust.  
+pitch angle measures the desired collective blade pitch that corresponds to zero thrust.
 
+.. note:: If the minimum or maximum collective is measured in ACRO (see below for impact of ACRO on collective pitch curve vs stabilized modes), the value will be less than those since some servo movement is reserved for cyclic. Of course, any swashplate tilt due to pilot or autopilot input will interefere with the measurements also.
```

Comment:
What is this supposed to mean? “The value will be less than those”. The value of what?  I believe in the SV_MAN, cyclic is locked out in min, mid and max positions. Why does cyclic factor into this discussion?

Reply:
if H_SRV_MAN =0 then you will not get max and min collective at the extremes of stick movement  in ACRO, the only mode that has full collective range, irrespective of the IM_STB_COL params...will make it clearer

Reply:
Well they shouldn’t be measuring collective blade pitch with SV_MAN = 0. I think you should just state that measurement of collective blade pitch will not be accurate with SV_MAN disabled because the autopilot is making swashplate inputs according to control laws or something to that end

Reply:
will do...